avm/res/databricks/workspace/tests/e2e/max/dependencies.bicep

```diff
@@ -94,7 +94,7 @@ resource keyPermissions 'Microsoft.Authorization/roleAssignments@2022-04-01' = {
   name: guid('msi-${keyVault::key.id}-${location}-${managedIdentity.id}-Key-Key-Vault-Crypto-User-RoleAssignment')
   scope: keyVault::key
   properties: {
-    principalId: 'b9199ba0-d7a7-499e-a9c1-ee87112c5aec' // AzureDatabricks Enterprise Application Object Id (Note: this is tenant specific)
+    principalId: '711330f9-cfad-4b10-a462-d82faa92027d' // AzureDatabricks Enterprise Application Object Id (Note: this is tenant specific)
```

Comment:
too bad

Reply:
Too bad indeed. This is the AVM one (hence the pipeline is red).
Mind you - I did try to fetch the ID via an MSI+DS but as expected it doesn't have the permissions to do that.